tests/unittest.py

```diff
@@ -76,10 +74,6 @@ def __init__(self, methodName, *args, **kwargs):
 
         @around(self)
         def setUp(orig):
-            # enable debugging of delayed calls - this means that we get a
-            # traceback when a unit test exits leaving things on the reactor.
-            twisted.internet.base.DelayedCall.debug = True
```

Comment:
I seem to remember that this was important?

Reply:
@erikjohnston and I decided to get rid of it -- everything using HomeserverTestCase shouldn't run into the problems that caused this to be enabled (plus it slows down the tests a LOT).